docker/nginx/dev/Dockerfile

```diff
@@ -3,6 +3,6 @@ FROM nginx:${NGINX_IMAGE_VERSION}
 
 RUN apk add openssl bash
 
-ADD docker/nginx/dev/startup.sh /opt/startup.sh
+ADD startup.sh /opt/startup.sh
```

Comment:
Please undo this

Reply:
Didn't work for me without that, in docker-compose.dev.yml the `context` for the nginx dockerfile is already `.docker/nginx/dev`
We could make it more consistent by just removing the `context` line and then restoring the `docker/nginx/dev` prefix here

Reply:
Oh, sorry, my fault, let's do as you said, restore this, and change in `docker-compose.dev.yml` nginx section to

```
        build:
            context: .
            dockerfile: docker/nginx/dev/Dockerfile
            args:
                - NGINX_IMAGE_VERSION=${NGINX_IMAGE_VERSION}
```